src/Angor/Client/Services/FeatureFlagService.cs

```diff
@@ -1,15 +1,18 @@
 using Angor.Client.Storage;
+using Angor.Shared;
 
 namespace Angor.Client.Services;
 
 public class FeatureFlagService : IFeatureFlagService
 {
     private readonly IClientStorage _storage;
-    private readonly Dictionary<string, bool> _featureFlags;
-    public FeatureFlagService(IClientStorage storage)
+    private readonly INetworkConfiguration _networkConfiguration;
+    private Dictionary<string, bool> _featureFlags;
+    public FeatureFlagService(IClientStorage storage, INetworkConfiguration networkConfiguration)
     {
         _storage = storage;
-        _featureFlags = _storage.getFeatureFlags();
+        _networkConfiguration = networkConfiguration;
+        _featureFlags = _storage.getFeatureFlags() ?? [];
```

Comment:
As a rule try to not do anything heavy in contractors, they are called by the service dependency injector and should be as light weight as possible.
you can move this code to the `GetAllFeatureFlags()` method like `_featureFlags ?? _storage.getFeatureFlags()`